ui/public/javascript/editForm.js

```diff
@@ -121,6 +121,14 @@ Mousetrap.bind("l", function() {
     document.querySelector(".ingredients-list:last-child").scrollIntoView({
         behavior: 'smooth'
     });
+}
+
+//try to parse a whole block
+Mousetrap.bind("l", function() {
```

Comment:
Looks like you can assign multiple keys to function, see: https://craig.is/killing/mice

So:

Mousetrap.bind("l", "L", function() {
   parseWholeBlockOfRecipeTextForIngredients()
})